packages/block-editor/src/components/plain-text/index.native.js

```diff
@@ -41,16 +41,13 @@ export default class PlainText extends Component {
 			<TextInput
 				{ ...this.props }
 				ref={ ( x ) => this._input = x }
-				className={ [ styles[ 'block-editor-plain-text' ], this.props.className ] }
 				onChange={ ( event ) => {
 					this.props.onChange( event.nativeEvent.text );
 				} }
 				onFocus={ this.props.onFocus } // always assign onFocus as a props
 				onBlur={ this.props.onBlur } // always assign onBlur as a props
-				fontFamily={ this.props.fontFamily || ( styles[ 'block-editor-plain-text' ].fontFamily ) }
-				fontSize={ this.props.fontSize }
-				fontWeight={ this.props.fontWeight }
-				fontStyle={ this.props.fontStyle }
+				fontFamily={ this.props.style.fontFamily || ( styles[ 'block-editor-plain-text' ].fontFamily ) }
```

Comment:
Wouldn't `this.props.style.fontFamily` throw an error if `this.props.style === undefined`?

Reply:
Thanks for catching this, I fixed it here: https://github.com/WordPress/gutenberg/pull/18785/commits/dd461084d3f169dfed5b728ad00f5614fefccdf2